test/Data.purs

```diff
@@ -552,8 +552,9 @@ instance (FromData a) => FromData (Tree a) where
   fromData x = genericFromData x
 
 fromBytesFromData :: forall a. FromData a => String -> Maybe a
-fromBytesFromData binary = fromData =<< PDD.convertPlutusData =<< fromBytes
-  (hexToByteArrayUnsafe binary)
+fromBytesFromData binary = fromData =<< (Just <<< PDD.convertPlutusData) =<<
```

Comment:
Can use `map` or `<$>` instead of `(Just <<< ...)`

Reply:
For cases like this, you can just compose

```suggestion
fromBytesFromData binary = fromData <<< PDD.convertPlutusData =<<
```